docs/farming.md

```diff
@@ -215,10 +215,16 @@ services:
 # Replace `INSERT_YOUR_ID` with your node ID (will be shown in telemetry)
       "--name", "INSERT_YOUR_ID"
     ]
+    healthcheck:
+      timeout: 5s
+# If node setup takes longer then expected, you want to increase `interval` and `retries` number.
+      interval: 30s
+      retries: 5
```

Comment:
As I said before, we can't specify this before we publish updated images. Are you sure this still works fine without healthcheck defined in current images?

Reply:
I thought yes, but discovered that it will fail as there is no healthcheck 😅. Should I make this pr draft then, so we could merge it right before the release?

Reply:
Yes, there will be more documentation updates before we make the next release anyway